src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -825,8 +852,12 @@ import './Menu.css'
 | `.pf-c-menu__item-action` | `<button>` | Initiates the menu item action. |
 | `.pf-c-menu__item-action-icon` | `<span>` | Initiates the menu item action icon. |
 | `.pf-c-menu__item-external-icon` | `<span>` | Initiates the menu item external icon. |
-| `.pf-m-submenu` | `.pf-c-menu` | Modifies the menu to be the submenu. |
 | `.pf-m-favorite` | `.pf-c-menu__item-action` | Modifies the menu item action to handle the favorite icon. |
 | `.pf-m-favorited` | `.pf-c-menu__item-action.pf-m-favorite` | Modifies the menu item action icon to be favorited. |
 | `.pf-m-selected` | `.pf-c-menu__item` | Modifies the menu item to be selected. |
-| `.pf-m-flyout` | `.pf-c-menu` | Modifies the menu so that all nested `.pf-c-menu` elements "flyout" |
+| `.pf-m-flyout` | `.pf-c-menu` | Modifies the menu so that all nested `.pf-c-menu` elements "flyout". |
+| `.pf-m-drilldown` | `.pf-c-menu` | Modifies the menu so that all nested `.pf-c-menu` elements "drill down". |
+| `.pf-m-current-path` | `.pf-c-menu.pf-m-drilldown .pf-c-menu__list-item` | Modifies the menu list item for current path state. |
+| `.pf-m-drilled-in` | `.pf-c-menu.pf-m-drilldown, .pf-c-menu.pf-m-drilldown .pf-c-menu` | Modifies the menu list for drilled in state. |
+| `--pf-c-menu--m-drilldown--Width: {width}` | `.pf-c-menu.pf-m-drilldown` | Modifies the width of the drilldown menu. The default value is `auto`. |
```

Comment:
should this just be a variable ? and maybe a modifier that specifies that you are modifying the width

Reply:
`--pf-c-menu--m-drilldown--Width` is set to auto be default, so the menu will be as wide as the first parent list. All nested menus will also be as wide as the first parent list because it provides the background/drop shadow styling. The width of the dropdown can be updated inline by modifying `--pf-c-menu--m-drilldown--Width`. Using this method means that devs can use any width value they want, rather than relying on a predefined class.